tests/Feature/Http/Controllers/HomeControllerTest.php

```diff
@@ -0,0 +1,665 @@
+            ->where('staticData.totalPointsEarned', $staticData->TotalPointsEarned)
+        );
+    }
+
+    public function testItCorrectlySendsMostRecentMasteryProps(): void
+    {
+        // Arrange
+        $system = System::factory()->create();
+        $game = Game::factory()->create([
+            'ConsoleID' => $system->id,
+        ]);
+        $player = User::factory()->create();
+
+        StaticData::factory()->create([
+            'last_game_hardcore_mastered_game_id' => $game->id,
+            'last_game_hardcore_mastered_user_id' => $player->id,
+        ]);
+
+        // Act
+        $response = $this->get(route('demo.home'));
+
+        // Assert
+        $response->assertInertia(fn (Assert $page) => $page
+            ->where('mostRecentGameMastered.game.id', $game->id)
+            ->where('mostRecentGameMastered.game.title', $game->title)
+            ->where('mostRecentGameMastered.game.badgeUrl', $game->badgeUrl)
+
+            ->where('mostRecentGameMastered.game.system.name', $system->name)
+            ->where('mostRecentGameMastered.game.system.iconUrl', $system->iconUrl)
+
+            ->where('mostRecentGameMastered.user.displayName', $player->username)
+            ->where('mostRecentGameMastered.user.avatarUrl', $player->avatarUrl)
+        );
+    }
+
+    public function testItCorrectlySendsMostRecentGameBeatenProps(): void
+    {
+        // Arrange
+        $system = System::factory()->create();
+        $game = Game::factory()->create([
+            'ConsoleID' => $system->id,
+        ]);
+        $player = User::factory()->create();
+
+        StaticData::factory()->create([
+            'last_game_hardcore_beaten_game_id' => $game->id,
+            'last_game_hardcore_beaten_user_id' => $player->id,
+        ]);
+
+        // Act
+        $response = $this->get(route('demo.home'));
+
+        // Assert
+        $response->assertInertia(fn (Assert $page) => $page
+            ->where('mostRecentGameBeaten.game.id', $game->id)
+            ->where('mostRecentGameBeaten.game.title', $game->title)
+            ->where('mostRecentGameBeaten.game.badgeUrl', $game->badgeUrl)
+
+            ->where('mostRecentGameBeaten.game.system.name', $system->name)
+            ->where('mostRecentGameBeaten.game.system.iconUrl', $system->iconUrl)
+
+            ->where('mostRecentGameBeaten.user.displayName', $player->username)
+            ->where('mostRecentGameBeaten.user.avatarUrl', $player->avatarUrl)
+        );
+    }
+
+    public function testItCorrectlySendsAchievementOfTheWeekProps(): void
+    {
+        // Arrange
+        $system = System::factory()->create();
+        $game = Game::factory()->create([
+            'ConsoleID' => $system->id,
+        ]);
+
+        // TODO use event achievements
+        $achievement = Achievement::factory()->create([
+            'ID' => 9,
+            'Title' => 'That Was Easy',
+            'GameID' => $game->id,
+        ]);
```

Comment:
Just noting more things to update here when we cut over to using event achievements.